contracts/RelayHub.sol

```diff
@@ -195,7 +195,8 @@ contract RelayHub is IRelayHub {
 
     function relayCall(
         EnvelopingTypes.RelayRequest calldata relayRequest,
-        bytes calldata signature
+        bytes calldata signature,
```

Comment:
let's set a precedence here and let's use explicit types.
```suggestion
        bytes32 calldata signature,
```

Reply:
done

Reply:
@jurajpiar @ironFe93  Are we sure that it doesn't break anything? In my understanding they are two different types, and I don't remember the size of the signature, so I'm not sure if it fits in a bytes32.

Reply:
bytes == bytes32... it's just better to be expressive about size

Reply:
actually, after re-reading the docs, I see this is not the case :/
`bytes` type is tightly packed (so it may have a smaller footprint), whereas `bytesN` will aways take the N-bytes of space.